Fixed documentation! (#4958)
hammerandchisel/discord-api-docs: docs/Getting_Started.md

# Getting started with Discord App Development

> Discord applications let you customize your servers with interactions and automation. This guide is meant to walk through building and running your first Discord app using JavaScript. At the end of this guide, you’ll have an app with a bot user that uses slash commands, sends messages, and interacts with message components.

Reviewer: The use of applications vs apps is very inconsistent in this now

![Demo of example app](getting-started-demo.gif)

To make the user flow a bit more explicit:
1. User 1 initiates a new game and picks their object using the app’s `/challenge` slash command.
2. A message is sent to channel with a button inviting others to accept the challenge.
3. User 2 presses the **Accept** button to accept the challenge.
4. User 2 is sent an ephemeral message where they select their object of choice.
5. The result of the game is posted back into the original channel for all to see.

## Creating an app

> The first thing we’ll need to do is to create an app. Navigate to the [developer dashboard](https://discord.com/developers/applications), then click on the **New Application** Tab in the upper right corner.

Reviewer: That is not a tab

### Adding scopes and permissions

> Apps need approval from installing users to perform actions inside of their Discord server (like creating a slash command or adding emojis). So before installing your app, let's add some scopes and permissions to request during installation. Click on **OAuth2** in the left sidebar, then **URL Generator**.

Reviewer: Apps don't own servers, so this change is wrong

First, copy your bot user’s token from earlier and paste it in the **`DISCORD_TOKEN`** variable in your `.env` file.

> info

> > .env is such a file where all your sensitive crenditials and data stored, cannot be accessed by anyone even if it is public i.e. it remains safe within the user's environment.

Reviewer: ```suggestion
> .env is a file where all your sensitive crenditials and data is stored, which should not be accessible to anyone, even if it is public
```
